app/components/conversations/conversation-part-closed.js

```diff
@@ -0,0 +1,22 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { computed, get } from '@ember/object';
+import { alias } from '@ember/object/computed';
+
+export default Component.extend({
+
+  author: null,
+
+  classNames: ['conversation-part', 'conversation-part--closed'],
+  classNameBindings: ['isSelf:conversation-part-closed--is-self'],
```

Comment:
@joshsmith Your comment got marked as outdated. I still think it's needed for the "displaying it left vs right" part of the style.

@daveconnis This needs to be 

```Javascript
classNameBindings: ['isSelf:conversation-part--is-self'],
```

In case it's not clear, this tells Ember to add a `conversation-part--is-self` css class to the component's HTML element, whenever the `isSelf` property on the component is `true` (or truthy).

We then use that class to style component differently when it's displayed to the user. Like in any other chat interface, if it's your message, or your action, it's written on your side (left vs right), compared to when it's the other user's message, where it's written on their side.